app/views/publish/courses/study_mode/new.html.erb

```diff
@@ -1,10 +1,10 @@
-<% content_for :page_title, title_with_error_prefix("Full time or part time", @errors && @errors.any?) %>
+<% content_for :page_title, title_with_error_prefix(t("page_titles.study_mode.new"), @errors && @errors.any?) %>
 
 <% content_for :before_content do %>
   <%= govuk_back_link_to(@back_link_path) %>
 <% end %>
 
-<%= render "publish/shared/errors" %>
+<%= render "publish/shared/errors", error_id: "#course-study-mode-field-error" %>
```

Comment:
I don't understand why this is necessary to change. Does the existing error mechanism not work now or does it have something to do with accessibility?

Reply:
`publish/shared/errors` doesn't work with the govuk checkboxes fieldset. It expects something other than the default field error id, breaking the href navigation from the error summary box to the fieldset. I've used the `govuk_uk checkboxes` as they were intended, which I think is better than writing bespoke html. Ideally, they would be paired with the `govuk_error_summary` component here (as we do for the edit side of things). But that would require refactoring the `CreationService`, which includes some funky error handling, and that is beyond the scope of the work we're doing here.

Reply:
I've reverted the code which fixes the error navigation for this and created a ticket for fixing it everywhere.